### DISCUSSION TITLE
Ability to set asset focal point via GraphQL

### DISCUSSION THREAD

**Discussion by robinbeatty** [craftcms/cms#13846] [Ideas] [answered]
Probably an edge case, but would love to be able set/update asset focal point values via GraphQL. Thanks

**Comment by brandonkelly** [comment, marked as answer]
Easy enough! Added for the next release (35fa59f0bc39789e0e10c6ba3e38dc11c751c014).

**Reply by robinbeatty** [reply]
Hero! Many thanks!

**Reply by brandonkelly** [reply]
Craft 4.5.8 is out now with that new asset mutation field 🎉
